third-party/test/index.ts

```diff
@@ -118,8 +118,12 @@ function run(testsRoot, clb): any {
         coverageRunner.setupCoverage();
     }
 
+    // 2020-03-24: Amazon addition.
+    const testFile = process.env["TEST_FILE"]?.replace(/^src\/test\//, "")?.concat('.js')
+    // END 2020-03-24: Amazon addition.
```

Comment:
Nit: move the Amazon addition to after the `glob`

Reply:
Good catch, this is more than a nit, since we should mark our divergent changes within the third-party folder

Reply:
Is there a reason we're doing this? MIT license doesn't require that you state your changes. It just requires that you include the MIT license.

Was this for our own purposes?